src/renderer/components/Tabs/Tabs.tsx

```diff
@@ -120,7 +127,7 @@ function Tabs() {
       url: string,
       httpResponseCode: number
     ) => {
-      if (url.includes('/accounts/SetSID?') && httpResponseCode === 400) {
+      if (url.includes('/accounts/SetSID') && httpResponseCode === 400) {
```

Comment:
This error page does not have params on Windows, so '/accounts/SetSID?' never matched